mapswipe_workers/tests/unittests/tear_down.py

```diff
@@ -0,0 +1,43 @@
+"""Helper functions for test tear down"""
+
+from mapswipe_workers import auth
+
+
+def delete_test_data(project_id: str) -> None:
```

Comment:
should we double check that project id is not null or "" here? or are you sure about it?

Reply:
Yeah. We probably should. 

Added a check!